bin/ebash-repl

```diff
@@ -80,7 +80,7 @@ die_handler()
 }
 
 $(opt_parse \
-    "=load l | Load the specified file prior to running the interactive interpreter.")
+    ":load l | Load the specified file prior to running the interactive interpreter.")
```

Comment:
I hadn't meant to commit this in the last PR. And I didn't notice this was modified until I used it to validate some things in this PR.